File: Swift-C++-Testsuite/AutoTranspiledTests/try-0/FunctionVariadicParameter.cpp
 
#include <iostream>
#include <vector>

using namespace std;

double arithmeticMean(vector<double> numbers) {
    double total = 0;
    for (int i = 0; i < numbers.size(); i++) {
        total += numbers[i];
    }
    return total / numbers.size();
}

int main() {
    vector<double> values = {3, 8.25, 18.75};
    double res = arithmeticMean(values);
    cout << res << endl;
    
    return 0;
}

File: Swift-C++-Testsuite/test/functions/Swift/FunctionVariadicParameter.swift
@main
struct MyMain {

    static func main() {
        let res = arithmeticMean(3.2, 8.25, 18.75)
    }

}

func arithmeticMean(_ numbers: Double...) -> Double {
    var total: Double = 0
    for number in numbers {
        total += number
    }
    return total / Double(numbers.count)
}
